### DISCUSSION TITLE
Difficulties running IEA-15-240-RWT-UMaineSemi model

### DISCUSSION THREAD

**Discussion by Uisliu** [NatLabRockies/ROSCO#307] [Q&A]
Hi all,
I am looking for some assistance regarding simulations that I am trying to conduct for the IEA-15-240-RWT-UMaineSemi model So far I have;
Installed Openfast (Precompiled binaries)
Installed the IEA-15-240-RWT-UMaineSemi master file
Installed ROSCO (Precompiled along with the libdiscon.dll file)
When running the simulation Through OpenFAST I got the following errors;
 
![image](https://github.com/NREL/ROSCO/assets/71447919/12843017-211e-4526-8258-3fc8f9d98004)

These errors seem to suggest that I am missing files of some sort. Could this be a problem with versions or am I missing a program altogether? 
Looking forward to some input : )

**Comment by dzalkind** [comment]
Hi, it looks like you are using an old input file with the latest version of ROSCO.  Based on the last line, it looks like things are running.

Dan

**Reply by Uisliu** [reply]
Hi dzalkind, Thank you very much for the response. I downloaded the latest version of ROSCO, libdiscon.dll, OpenFAST and the IEA 15MW turbine FAST files and received the same errors. The current version of ROSCO that I am using is version ROSCO v2.9.0. 
The IEA-15-240-RWT-UMaineSemi_DISCON.IN file mentions that I was made using ROSCO v2.8.0. Does this mean that I need the libdicson.dll file from the ROSCO 2.8.0 to run the simulation? When looking through the ROSCO github page I could not find the ROSCO 2.8.0 version, would you possibly know where I might fine it?
  
![image](https://github.com/NREL/ROSCO/assets/71447919/a71d9512-c1b0-4f55-950f-c3d46cc75901)

Thanks again for the reply on this post.

**Reply by Uisliu** [reply]
Hi dzalkind, I was able to locate the ROSCO 2.8.0 dll file and those errors disappeared! I will probably need more assistance at some point though😅.
